run.zsh

```diff
@@ -166,10 +189,13 @@ if ( ! ${keep_frameworks} ); then
 fi
 
 # cleanup any corpses/leftovers
-if ( ! ${has_zplug} ); then
-  rm -rf ${ZDOTDIR:-${HOME}}/.zplug
-fi
+if ( ${force_delete} ); then
```

Comment:
Some people may have zplug / OMZ installed, so removing these files is not a safe default.

Fortunately zplug has fixed `ZPLUG_HOME` in the current version, so at least that part can be removed.